### DISCUSSION TITLE
Transparent Menu

### DISCUSSION THREAD

**Discussion by Pinox** [microsoft/fluentui-blazor#1614] [General]
When I think about fluent-ui then transparent menu is the first thing that comes to mind.

![image](https://github.com/microsoft/fluentui-blazor/assets/5297255/bed0d2b8-e503-4a6b-ac1b-1110bd46e643)

Will something like this ever be possible within browsers / fluent-ui ?

**Comment by vnbaaij** [comment]
Yes, I think this is definitely possible in a browser. Just not something I see us implementing any time soon as it is rather specific. We offer the NavMenu (in use in the demo site) and soon the AppBar as you know from Teams:
![FluentAppBar 2](https://github.com/microsoft/fluentui-blazor/assets/1761079/e8c0183e-0248-4722-ae9d-020755a46897)
